include/seqan3/alphabet/concept.hpp

```diff
@@ -215,7 +215,7 @@ struct assign_rank_to_cpo : public detail::customisation_point_object<assign_ran
         /*return*/ static_cast<alphabet_t>(seqan3::custom::alphabet<alphabet_t>::assign_rank_to(rank, alphabet)) /*;*/
     );
 
-    /*!\brief CPO overload (2. out of 3 checks): argument dependent lookup (ADL), i.e. `assign_rank_to(rank, alphabet)`
+    /*!\brief CPO overload (check 2 out of 3 checks): argument dependent lookup (ADL), i.e. `assign_rank_to(rank, alphabet)`
```

Comment:
```suggestion
    /*!\brief CPO overload (check 2 out of 3): argument dependent lookup (ADL), i.e. `assign_rank_to(rank, alphabet)`
```

Also has the advantage that the lines are not longer than 120